nuxeo-coldstorage/src/test/java/org/nuxeo/coldstorage/operations/RequestRetrievalFromColdStorageTest.java

```diff
@@ -61,6 +61,8 @@ public void shouldRequestRetrieval() throws OperationException, IOException {
         DocumentModel documentModel = createFileDocument(session, true);
         // first make the move to cold storage
         moveContentToColdStorage(session, documentModel);
+        //Mock AWS Lifecycle rule
+        ColdStorageS3TestHelper.updateObjectClassStorage(session.getDocument(documentModel.getRef()));
```

Comment:
I think instead of duplicating of the call to 
`ColdStorageS3TestHelper.updateObjectClassStorage(session.getDocument(documentModel.getRef()));`

We can move it the abstract class test. With java comment to explain as you did the reason of the manual move/copy/class storage update